src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+
+        // initialise an array of tasks that is <=100
+        //sizeOfList is the number of items in the list
+        Task[] listOfTasks = new Task[100];
+        int sizeOfList = 0;
+
+        // Reads in the first command from the user
+        Scanner scanner = new Scanner(System.in);
+        String inputCommand = scanner.nextLine();
+
+        while (inputCommand.equals("bye") == false) {
+
+            if (inputCommand.equals("list")) {
+                // prints out the list
+                printsOutTheList(listOfTasks, sizeOfList);
+
+            } else if (inputCommand.contains("done ") == true) {
+                // marks task in the stated index as done
+                indicateTaskAsDone(listOfTasks, inputCommand);
+
+            } else {
+                // adds a new task into the list
+                sizeOfList = addInNewTask(listOfTasks, sizeOfList, inputCommand);
+
+            }
+            // take in the next command
+            inputCommand = scanner.nextLine();
+        }
+
+        printExitMessage();
+    }
+
+    /**
+     * Prints the exit message
+     */
+    private static void printExitMessage() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Adds in new task to the array,
+     * returns the new size of the list
+     *
+     * @param listOfTasks array of tasks
+     * @param sizeOfList number of tasks in list
+     * @param inputCommand name of the task to add
+     * @return the updated number of tasks in list
+     */
+    private static int addInNewTask(Task[] listOfTasks, int sizeOfList, String inputCommand) {
+        // creates new task
+        Task newTask = new Task(inputCommand);
+        listOfTasks[sizeOfList] = newTask;
+        // informs user that task has been added to list
+        System.out.println("____________________________________________________________");
+        System.out.println(" added: " + inputCommand);
+        System.out.println("____________________________________________________________");
```

Comment:
I think these three lines can be extracted to a method called something like "showFeedback"